Resources/Prototypes/Entities/Structures/Machines/vending_machines.yml

```
@@ -1637,8 +1637,8 @@
       shader: unshaded
     - state: panel
       map: ["enum.WiresVisualLayers.MaintenancePanel"]
-  - type: AccessReader
-    access: [["Service"]]
+#  - type: AccessReader # Frontier
```

```suggestion
  - type: AccessReader
```
Commenting out the `AccessReader` component entirely is somewhat of bad practice, since it renders the object incompatible with things like the Access Configurator (even if in this case such would be completely useless). Removing the `access` field is sufficient for this sort of thing.

---

Its a public vending machine you do not want poeple to lock it